src/Html/HtmlCssSelectors.fs

```diff
@@ -51,6 +51,14 @@ module internal HtmlCssSelectors =
         let getOffset (t:List<char>) = 
             charCount - 1 - t.Length
 
+        let isCharacterEscapable (c:char) =
+            (* CSS 2.1: Any character (except a hexadecimal digit, linefeed,
+               carriage return, or form feed) can be escaped with a backslash to
+               remove its special meaning *)
+            let isHexadecimalDigit = Char.IsDigit(c) || (Char.ToLower(c) >= 'a' && Char.ToLower(c) <= 'f')
+            (isHexadecimalDigit || c = '\n' || c = '\f' || c = '\r')
```

Comment:
I see .Equals being used instead below, is there an advantage in doing that? If I do have to change it to .Equals what about the hexadecimal digit comparison above where I need the ascii code ?